docs/core/tools/local-tools-how-to-use.md

```diff
@@ -0,0 +1,202 @@
+---
+title: "Tutorial: Install and use .NET Core local tools"
+description: Learn how to install and use a .NET tool as a local tool.
+ms.date: 02/12/2020
+---
+
+# Tutorial: Install and use a .NET Core local tool using the .NET Core CLI
+
+**This article applies to:** ✔️ .NET Core 2.1 SDK and later versions
```

Comment:
Changed this one to 3.0 here in the heading, the others are 2.1 in the heading with "available since 3.0" as needed in the body when referring to local tools.